packages/xo-server/src/xapi/mixins/networking.js

```diff
@@ -25,6 +25,19 @@ export default {
   async disconnectVif(vifId) {
     await this._disconnectVif(this.getObject(vifId))
   },
+  async _editNetwork(
+    id,
+    { automatic, defaultIsLocked, nameDescription, nameLabel, otherConfig }
```

Comment:
Are all this parameters mandatory?

Reply:
Yes, because we can edit all of this. No ?